api/src/utils/mailer-templates.ts

```diff
@@ -0,0 +1,169 @@
+                <br>
+                <dt>Thank you for submitting your pitch <a href='"${
+                  pitch.assignmentGoogleDocLink
+                }"'>"${
+    pitch.title
+  }"</a> Unfortunately, your pitch was declined. </dt>
+                If you have any questions or need any additional support, please contact ${
+                  admin.email
+                }. 
+                Check out more <a href='www.google.com'>pitch-writing resources</a>. In the meantime, feel free to check the 
+                  <a href='http://south-side-weekly.vercel.app/pitches'>pitch doc</a> for potential new stories to claim!</dt>
+                <br>
+                <dt>Thank you,
+                    <br>
+                ${admin.preferredName || admin.firstName}
+                </dt>
+                </div>`,
+});
+
+export const approvedMessage = (
+  author: IUser,
+  pitch: IPitch,
+  admin: IUser,
+): EmailMessage => ({
+  to: author.email,
+  from: process.env.EMAIL_USERNAME,
+  subject: `Pitch "${pitch.title}" Approved`,
+  html: `<div>
+                <dt>Hi ${author.preferredName || author.firstName},</dt>
+                <br>
+                <dt>Congratulations, your pitch <a href='"${
+                  pitch.assignmentGoogleDocLink
+                }"'>"${
+    pitch.title
+  }"</a> has been approved! Here are the details you provided regarding the pitch:</dt>
+                <br>
+                <b>Description:</b> ${pitch.description}
+                <br>
+                <b>Link:</b> ${pitch.assignmentGoogleDocLink}
+                <br>
+                <br>
+                <dt>Your pitch is due on ${
+                  pitch.deadline ? pitch.deadline.toDateString() : 'no date'
+                }. 
+                If you have any questions or need any additional support, please contact ${
+                  admin.email
+                }. 
+                We can't wait to see your story come together!</dt>
+                <br>
+                <dt>Thank you,
+                    <br>
+                ${admin.preferredName || admin.firstName}
+                </dt>
+                </div>`,
+});
+
+export const approveClaim = async (
+  author: IUser,
+  pitch: IPitchAggregate,
+  admin: Partial<IUser>,
+  teams: [string],
+): Promise<EmailMessage> => {
+  const teamNamesToUsers: teamsUsers = {};
+  for (const user of pitch.assignmentContributors) {
+    for (const teamName of user.teams) {
+      const userModel = await User.findById(user.userId);
+      const info: userInfo = {
+        name: `${userModel.firstName} ${userModel.lastName}`,
+        email: userModel.email,
+      };
+      if (!(teamName in teamNamesToUsers)) {
+        teamNamesToUsers[teamName] = [info];
+      } else {
+        teamNamesToUsers[teamName].push(info);
+      }
+    }
+  }
+
+  const m = {
+    to: author.email,
+    from: process.env.EMAIL_USERNAME,
+    subject: `Claim Request for "${pitch.title}" Approved`,
+    html: `<html>
```

Comment:
Ideally this HTML should be abstracted and used as a variable here to be cleaner